src/renderer/viz/expressions/classification/Classifier.js

```diff
@@ -59,6 +69,23 @@ export default class Classifier extends BaseExpression {
         // TODO
         super._preDraw(program, drawMetadata, gl);
     }
+
+    getLegendData () {
+        const breakpoints = this.getBreakpointList();
+        const breakpointsLength = breakpoints.length;
+        const name = this.toString();
+        const data = [];
+
+        for (let i = 0; i <= breakpointsLength; i++) {
+            const min = i - 1 >= 0 ? breakpoints[i - 1] : Number.NEGATIVE_INFINITY;
```

Comment:
if it's easier, it could be replaced by `breakpoints[i-1] || Number.NEGATIVE_INFINITY`

Reply:
and same with max: breakpoints[i] || Number.POSITIVE_INFINITY?